tests/bluetooth/test_characteristic_emulation.py

```diff
@@ -314,3 +331,249 @@ async def test_bluetooth_remove_characteristic_uuid_with_properties(
                     'type': 'remove'
                 }
             })
+
+
+@pytest.mark.asyncio
+@pytest.mark.parametrize('capabilities', [{
+    'goog:chromeOptions': {
+        'args': ['--enable-features=WebBluetooth']
+    }
+}],
+                         indirect=True)
+@pytest.mark.parametrize(
+    'write_type', [('writeValueWithoutResponse', 'write-without-response'),
+                   ('writeValueWithResponse', 'write-with-response')])
+async def test_bluetooth_characteristic_write_event(websocket, context_id,
+                                                    html, write_type):
+    await setup_characteristic(websocket, context_id, html,
+                               HEART_RATE_SERVICE_UUID,
+                               DATE_TIME_CHARACTERISTIC_UUID, {'write': True})
+    await subscribe(websocket, ['bluetooth.characteristicEventGenerated'])
```

Comment:
nit: it can be helpful to move `bluetooth.characteristicEventGenerated` to a constant

Reply:
Done.